packages/nodejs/tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+    "extends": "../../tsconfig-base.json",
+    "include": ["src/**/*"],
+    "exclude": ["node_modules", "grpc/**/*"],
+    "rerefences": [
+        { "path": "../common"}
```

Comment:
I assume that it uses the outDir. (At least that seems to be the behaviour)
 
 > Where will the common stuff be accessible for a consumer of the published module?
 That is something to be decided on, I guess. Currently I just reexported everything from common, so that we could update without a user noticing.

Reply:
Yes, I think I was thinking of a case where `common` wouldn't be published as a separate package or something.. Makes sense now at least! :smile: